tests/StacktraceTest.php

```diff
@@ -39,15 +39,17 @@ public function testGetFramesAndToArray(): void
     {
         $stacktrace = new Stacktrace($this->options, $this->serializer, $this->representationSerializer);
 
-        $stacktrace->addFrame('path/to/file', 1, ['file' => 'path/to/file', 'line' => 1, 'function' => 'test_function']);
-        $stacktrace->addFrame('path/to/file', 2, ['file' => 'path/to/file', 'line' => 2, 'function' => 'test_function', 'class' => 'TestClass']);
+        $stacktrace->addFrame('path/to/file', 1, ['file' => 'path/to/file', 'line' => 2, 'class' => 'CrashyClass']);
+        $stacktrace->addFrame('path/to/file', 2, ['file' => 'path/to/file', 'line' => 1, 'function' => 'test_function']);
```

Comment:
It doesn't really matter given that this is a test, but for consistency the value of the line item in the 3rd argument should equal the value of the 2nd parameter as they represents the same thing which is the line at which the error originated